packages/scaffolding-cli/templates/src/ssr/package-lock.json

```diff
@@ -19901,9 +20575,9 @@
       }
     },
     "typescript": {
-      "version": "3.8.3",
-      "resolved": "https://registry.npmjs.org/typescript/-/typescript-3.8.3.tgz",
-      "integrity": "sha512-MYlEfn5VrLNsgudQTVJeNaQFUAI7DkhnOjdpAp4T+ku1TfQClewlbSuTVHiA+8skNBgaf02TL/kLOvig4y3G8w==",
+      "version": "3.9.3",
```

Comment:
Can we remove the the pinned version from README and package.json?

Reply:
Updated the readmes